### DISCUSSION TITLE
Towny War Question

### DISCUSSION THREAD

**Discussion by FizzleTits** [TownyAdvanced/Towny#4718] [Q&A] [answered]
I am trying to locate Rewards for War in the Towny configs.
I found where to set how many points for each kill, and plot capture.
I cannot find where to set the reward money for the winner of war.
Any help is much appreciated

**Comment by LlmDl** [comment, marked as answer]
The war spoils are populated by various ways in which Towny removes money from the economy, (like if a town is deleted and there's still money in the bank.) There's also one config option for the base spoils which are added at the beginning of the war, (added to whatever has already accumulated.)

**Reply by FizzleTits** [reply]
I used that config option to add 100k and the winner of war still gets 0

**Reply by LlmDl** [reply]
File a [bug ticket.](https://github.com/TownyAdvanced/Towny/issues/new?assignees=&labels=&template=bug_report.md&title=)

**Reply by FizzleTits** [reply]
But there isn't an error on console, only this: 
![image](https://user-images.githubusercontent.com/48837943/106380498-5ea48e00-6378-11eb-9920-0a8259fe4d65.png)

**Reply by LlmDl** [reply]
Chances are your economy plugin isn't working. File a [bug ticket.](https://github.com/TownyAdvanced/Towny/issues/new?assignees=&labels=&template=bug_report.md&title=)
